docs/tutorials/01_introductory/viz_keyframe_animation_minimal.py

```diff
@@ -0,0 +1,63 @@
+"""
+=====================
+Keyframe animation
+=====================
+
+Minimal tutorial of making keyframe-based animation in FURY.
```

Comment:
In this example, the arrow seems to disappear in my tests. This may be due to clipping boundaries problem.

Reply:
Yes, sometimes I run into this issue mostly when I interact with the show manager. and it's because of the clipping boundaries. Resetting the clipping plans with each update slows down the animation. I will try to solve it before our meeting.